src/config.ts

```diff
@@ -54,6 +39,72 @@ export interface TrussConfig {
    */
   fonts: FontConfig;
 
+  /** The design system's increment in pixels. */
+  increment: number;
+
   /** The number of increments to generate for rules like `mt1`, `mt2`, etc. */
   numberOfIncrements: number;
+
+  /** Short-hand aliases like `bodyText` --> `["f12", "black"]`. */
+  aliases?: Aliases;
+
+  /** Type aliases for Only clauses, i.e. `Margin` --> `["marginTop", ...]`. `Margin` and `Padding` are provided. */
+  typeAliases?: Record<string, Array<keyof Properties>>;
+
+  /** Breakpoints, i.e. `{ sm: 0, md: 500 }`. */
+  breakpoints?: Record<string, number>;
+
+  /**
+   * A map of "section" to list of rules to create utility methods.
+   *
+   * I.e. "borderColors" -> () => [`get ml1() { ... }`].
+   *
+   * This can be used to either add new sections or override built-in sections.
+   */
+  sections?: Record<SectionName, RuleFn>;
```

Comment:
I'm starting to get cute with type aliases, i.e. this `SectionName` is just a string, but helps some readability vs "um, everything is a string I guess?".

`UtilityMethod` is the another nice imo type alias that is added here.